packages/canary-web-components/src/components/ic-date-picker/ic-date-picker.tsx

```diff
@@ -1299,10 +1370,17 @@ export class DatePicker {
     return inputProps;
   };
 
-  render() {
+  renderHiddenInput = () =>
```

Comment:
```suggestion
  private renderHiddenInput = () =>
```

needed to resolve stenciljs lint error

Reply:
Fixed.